src/bp/core/services/telemetry/actions.ts

```diff
@@ -0,0 +1,111 @@
+import { parseActionInstruction } from 'common/action'
+import { BUILTIN_MODULES } from 'common/defaults'
+import LicensingService from 'common/licensing-service'
+import { getSchema } from 'common/telemetry'
+import Database from 'core/database'
+import { calculateHash } from 'core/misc/utils'
+import { TelemetryRepository } from 'core/repositories/telemetry_payload'
+import { TYPES } from 'core/types'
+import { inject, injectable } from 'inversify'
+import ms from 'ms'
+import path from 'path'
+
+import { GhostService } from '..'
+import { JobService } from '../job-service'
+
+import { TelemetryStats } from './telemetry-stats'
+
+type NextNode = {
+  condition: string
+  node: string
+}
+
+type Node = {
+  id: string
+  name: string
+  next: Array<NextNode>
+  onEnter: Array<string>
+  onReceive: Array<string>
```

Comment:
Please use native types: `onReceive: string[]`

Reply:
fixed in c69de5cf271e960e8deebbcc1c7b1abd0f3af91c